robocopkaka/bootcamp-29-project
# Feat delete events

#### What does this PR do?
Delete events
#### Description of Task to be completed?
* Add a `delete` button to the `EventsListWithImage` component
* Add a method for deleting event to the `Events` and `AdminProfile` components
* Add an onClick event handler that calls the `deleteEvent` method to the `EventsListWithImage` component
* Add action and action creator for deleting events
* Add method for deleting events to the EventApi class
* Add case for deleting events successfully to the event reducer
#### How should this be manually tested?
* Click on the delete icon on an event on the Admin profile
* The event should be removed from the DOM
* On page reload, the event should no longer be seen on the DOM
#### Any background context you want to provide?
Currently only works for Admins, and not regular users
#### What are the relevant pivotal tracker stories?(if applicable)
#155485293
